doc/port.1.txt

```diff
@@ -129,7 +129,7 @@ The port command recognizes several global flags and options.
     Quiet mode, suppress informational messages to a minimum, implies -N
 
 -N::
-    Non-interactive mode, interactive questions are not asked 
+    Non-question mode, interactive questions are not asked (avoid typing yes)
```

Comment:
I agree with Rainer, this phrasing is not good. As well as the behaviour not being the same as answering yes all the time, there are other types of question than just yes/no.